inst/doctest.m

```diff
@@ -160,6 +160,19 @@
 %% @end group
 %% @end example
 %%
+%% You can also mark tests depending on runtime conditions, such as the
+%% platform, operating systems, or installed packages:
+%%
+%% @example
+%% @group
+%% >> "some shiny octave feature"         % doctest: +XFAIL_IF(DOCTEST_MATLAB)
+%% ans = some shiny octave feature
+%% @end group
+%% @end example
+%%
+%% Doctest provides the default flags DOCTEST_OCTAVE and DOCTEST_MATLAB,
+%% but you can access arbitrary variables or functions.
+%%
```

Comment:
`but you can access arbitrary variables or functions.`  <-- is this accurate?

I.e., is this supposed to work?
`+SKIP_IF(compare_versions(ver('symbolic').Version, '2.2.2', '>='))`

That seems like a useful thing to be able to do.  I guess one can just store it as a variable.  (No need to hold up the merge for this)